packages/fm-simplex-plugin/fm-simplex-plugin.1.30/opam

```diff
@@ -24,7 +24,7 @@ remove:
 depends: [
   "ocaml" {>= "4.01.0"}
   "zarith" {>= "1.4"}
-  "camlzip"
+  "camlzip" {< "1.08"}
```

Comment:
You don't know that. On the tests they failed only because alt-ergo failed.

Reply:
Fair enough, but it is in a separate commit so I can check